parachains/runtimes/assets/statemint/src/lib.rs

```diff
@@ -829,13 +911,15 @@ impl_runtime_apis! {
 				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef70a98fdbe9ce6c55837576c60c7af3850").to_vec().into(),
 				// System Events
 				hex_literal::hex!("26aa394eea5630e07c48ae0c9558cef780d41e5e16056765bc8461851072c9d7").to_vec().into(),
+				// Configuration ActiveConfig
+				hex_literal::hex!("06de3d8a54d27e44a9d5ce189618f22db4b49d95320d9021994c850f25b8e385").to_vec().into(),
 			];
 
 			let mut batches = Vec::<BenchmarkBatch>::new();
 			let params = (&config, &whitelist);
 			add_benchmarks!(params, batches);
 
-			if batches.is_empty() { return Err("Benchmark not found for this pallet.".into()) }
+			// if batches.is_empty() { return Err("Benchmark not found for this pallet.".into()) }
```

Comment:
don't think batches should be empty?

Reply:
In which case does this error?  
I think you said that this change also happened somewhere else, do you remember where @ruseinov ?

Reply:
Here: https://github.com/paritytech/polkadot/pull/4442/files#diff-db5840c3c03c740de25aed56b9257559c224a04619c079c94cddbb23f9a855a4L1997

It fails when one runs xcm benches.

Reply:
@ggwpez ^

Reply:
I think it might have something to do with the namespace
`pallet_xcm_benchmarks::generic` vs `pallet_assets`, for example.

Reply:
removing it is correct

Reply:
https://github.com/paritytech/substrate/pull/9924

This PR has context.

Reply:
Then please delete the line completely.